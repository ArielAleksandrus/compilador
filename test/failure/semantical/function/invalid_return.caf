int f(){
    retorne 'a';
}
programa{}

