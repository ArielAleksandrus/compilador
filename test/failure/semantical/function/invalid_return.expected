ERROR => Line 2, col 15: Function type is 'int' but told to return 'car'
